source/framework/CMakeLists.txt

```diff
@@ -81,18 +81,7 @@ marlin_add_plugin_library( TARGETS CorePlugins )
 set( MarlinMT_BINARIES MarlinShellCompletion MarlinDumpPlugins Marlin MarlinMT )
 foreach( binary ${MarlinMT_BINARIES} )
   add_executable( bin_${binary} main/${binary}.cc )
-  target_include_directories( 
```

Comment:
@andresailer I guess you refereed to this. It compiles fine. And I agree, the different CMake implementation of dependencies is tricky. For example, DD4hep constructs the imported libraries for its dependencies (ROOT, LCIO, etc...) but I "find_package" ROOT only it doesn't setup the targets import.

Reply:
If you pick up the ROOTConfig.cmake from ROOT (above version 6.14) you get the `ROOT::<Targets>` with all the properties. If you get ROOT with some FindROOT.cmake that would interfere.

Reply:
I need to try again but my first try failed.

Reply:
Works like a charm now !